main.tf

```diff
@@ -1,7 +1,7 @@
 terraform {
   required_version = ">= 1.3.0, < 2.0.0"
   required_providers {
-    aws = ">= 4.0.0"
+    aws = ">= 5.26.0"
```

Comment:
I'm concerned that this range would allow versions that could cause errors. We should sort that out, which might be possible in a minor/patch release.